src/math-functions.js

```diff
@@ -82,7 +93,13 @@ to use the + operator for string concatenation.
 */
 
 export function sumArrayWithThreeNumbers(sumArr) {
+    const sumOne = sum(sumArr[0], sumArr[1][0]);
```

Comment:
sum(sumArr[0], sumArr[1][0]); should be sum(sumArr[0], sumArr[1])[0]; otherwise it's a NaN.